cohort_4/playbook/workshops/portfolio/github.md

```diff
@@ -0,0 +1,59 @@
+- Let's get this website on the real internet.
+  - [GitHub](https://github.com/) is where millions of developers store their
+    code online. One of their services,
+    [GitHub Pages](https://pages.github.com/), will put a website online for
+    you. We'll be using GitHub pages to host our portfolio.
+  - Head over to https://github.com/ and create a free account.
+
+    ![](img/github_create_account.png)
+
+  - Before proceeding, **verify the email on your GitHub accounts**. The
+    following steps _will not_ work if you don't do this. If you check your
+    inbox, you'll see an email from GitHub with a confirmation link.
+  - Once you have an account created, go to the GitHub homepage at
+    https://github.com and create a new repository with the name
+    _username_.github.io (replacing _username_ with your GitHub username).
+    Make sure `Initialize this repository with a README` **is not** checked.
```

Comment:
This is a random detail they shouldn't have to deal with. By default it is _not_ checked. Why are we telling them to make sure of this? Should we also make sure they don't check `License` or `.gitignore`?